server/datastore/mysql/software.go

```diff
@@ -373,8 +373,8 @@ func listSoftwareDB(
 			ids[result.ID] = idx
 		}
 
-		// handle null cve from left join
-		if result.CVE != nil {
+		// handle null cve from left join also skip any non-cve entries
+		if result.CVE != nil && strings.HasPrefix(strings.ToLower(*result.CVE), "cve") {
```

Comment:
Is not clear yet what we want to do we non-cve entries. Also, the number of non-cve entries should be very low, so applying the filtering at this level should be fine.

Reply:
Should non-cve entries even be in the `software_cve` table? I could see this approach having some issues eg should we return software when searching `query=USN`? should we return software when `vulnerable=true`?

Reply:
Thanks - Both are valid concerns, I'll change the approach.